bin/ghe-restore

```diff
@@ -271,7 +277,7 @@ fi
 # Update the remote status to "complete". This has to happen before importing
 # ssh host keys because subsequent commands will fail due to the host key
 # changing otherwise.
-trap "" EXIT
+trap "" EXIT 1 2 3 15
```

Comment:
@gnawhleinad: I did originally try this but got the following error upon running the script on my development box:

    trap: SIGINT: bad trap